ci/runtests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+
+source $HOME/miniconda/etc/profile.d/conda.sh
+conda activate test
+
+pip install -e .
```

Comment:
This is what's making our tests fail now and it's not necessary. Please remove it.